forms.py

```diff
@@ -0,0 +1,12 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, EmailField
+from wtforms.validators import DataRequired, Email, EqualTo, Length
+
+
+class LoginForm(FlaskForm):
+    username = StringField('UserName', validators=[DataRequired()])
+    password = PasswordField('Password', validators=[DataRequired(), Length(min=8)])
+    confirm_password = PasswordField('Confirm Password', validators=[DataRequired(), EqualTo('password')])
+    email = EmailField('Email', validators=[DataRequired(), Email()])
+
```

Comment:
Если добавить поле `SubmitField`.
```suggestion
    submit - SubmitField('Зарегистрироваться')
```